transformer_engine/pytorch/csrc/extensions/pybind.cpp

```diff
@@ -10,6 +10,12 @@
 #include "../extensions.h"
 
 PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
+  // Efficeint memory softmax cross entropy
+  m.def("cross_entropy_fwd_sum_exp_cuda", &cross_entropy_forward_sum_exp,
+        "Softmax Cross_entropy Forward Sum & Exp");
+  m.def("cross_entropy_fwd_mean_log_cuda", &cross_entropy_fwd_mean_log,
+        "Softmax Cross_entropy Forward Mean & Log");
+  m.def("cross_entropy_bwd_cuda", &cross_entropy_bwd, "Softmax Cross_entropy Backward");
```

Comment:
```suggestion
  // Efficeint memory softmax cross entropy
  m.def("cross_entropy_fwd_sum_exp_cuda", &cross_entropy_forward_sum_exp,
        "Softmax Cross_entropy Forward Sum & Exp",
        py::call_guard<py::gil_scoped_release>());
  m.def("cross_entropy_fwd_mean_log_cuda", &cross_entropy_fwd_mean_log,
        "Softmax Cross_entropy Forward Mean & Log",
        py::call_guard<py::gil_scoped_release>());
  m.def("cross_entropy_bwd_cuda", &cross_entropy_bwd, "Softmax Cross_entropy Backward",
        py::call_guard<py::gil_scoped_release>());
```

Reply:
Hi ksivaman, Thank you very much for taking the time to review the code for me！

I have already made the modifications now.